callback.js

```diff
@@ -0,0 +1,53 @@
+var http = require('http');
+var fse = require('fs-extra');
```

Comment:
@psinha20 Why you're importing the fs-extra package. It can be done without this package. Think about it.